ext/standard/tests/php_ini_scanned_files.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+string php_ini_scanned_files ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
```

Comment:
IMO there's no need to check for PHP versions < 5.3.0, because the test is supposed to be included in PHP 7.1 upward only.